src/main/java/hw_11/Task3.java

```diff
@@ -0,0 +1,46 @@
+package hw_11;
+
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class Task3 {
+    public static void main(String[] args) {
+
+        Queue<String> queue = new LinkedList<>();
+
+        queue.offer("red");
+        queue.offer("black");
+        queue.offer("white");
+        queue.offer("orange");
+        queue.offer("yellow");
+
+        System.out.println(queue);
+
+        String str = "";
+
+        if (!queue.isEmpty())
```

Comment:
please use { and } symbols - it is more readable

Reply:
fixed